src/components/InngestCommHandler.ts

```diff
@@ -637,6 +645,8 @@ export class InngestCommHandler<
       });
 
     this._isProd = actions.isProduction ?? isProd(env);
+    // If we're in production always skip.
+    this._skipDevServer = actions.isProduction ?? skipDevServer(env);
```

Comment:
Should this be checking the above `this._isProd` instead so that we get the value of `isProd(env)`?

```suggestion
    this._skipDevServer = this._isProd ?? skipDevServer(env);
```